Dev10/src/Outputer.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+
+/**
+ * This class is for outputting arrays.
+ */
+public class Outputer {
+  /**
+   * Outputting an array of arrays.
+   *
+   * @param arrays the array of arrays
+   */
+  public void outputArrays(double[][] arrays) {
+    for (int i = 0; i < arrays.length; i++) {
+      for (int j = 0; j < arrays.length; j++) {
+        System.out.print(arrays[i][j]);
+      }
+      System.out.println();
+    }
+  }
+
+  /**
+   * Outputting an new array.
```

Comment:
What does it mean new array? Who guarantees that the array that is passed to the method is new and not old?